frontend/src/app/features/details/summary/SummaryInfo.tsx

```diff
@@ -33,14 +50,58 @@ const SummaryInfo: React.FC<ISummaryInfo> = ({
       console.log('Approve Handler Not Provided');
     });
 
+  const CENTER_OF_BC: LatLngTuple = [53.7267, -127.6476];
```

Comment:
same here hard coded value .

Reply:
These are. the central coordinates in order to show the map in general